tests/integration/test_config.py

```diff
@@ -27,6 +27,62 @@ async def cluster(adcm_client: ADCMClient, cluster_bundle: Bundle) -> Cluster:
     return cluster
 
 
+async def test_invisible_fields(cluster: Cluster) -> None:
+    expected_error = KeyError
+
+    service = await cluster.services.get()
```

Comment:
should we be more specific in what service do we want to get? seems we bind test result to test data harder than we could. Or perhaps we consider we are not going to change test data at all in the future?

Reply:
Yes, it makes sense to specify required service since there are more than 1 test in this module, thanks!